### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

File: getting-started/data-setup.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ Install the LSST Science Pipelines
 
 If you haven't already, you'll need to install the LSST Science Pipelines.
 We recommend that you install the pre-built binary packages by following the instructions at :doc:`/install/newinstall`.
-This tutorial is intended to work with the |eups-tag-mono| tag of the ``lsst_distrib`` EUPS package.
+This tutorial was developed using the ``w_2021_33`` tag of the ``lsst_distrib`` EUPS package.
```
Comment (timj) [inline getting-started/data-setup.rst:29]: Maybe we can say that we expect it to work with newer releases (until such a time as it doesn't).

**Review by timj — APPROVED** [review]

File: getting-started/data-setup.rst (hunk 1 above)
Comment (timj) [reply getting-started/data-setup.rst:29]: My point being that we don't want to force people to downgrade to an older release just to run the tutorial if they don't need to.

Comment (SimonKrughoff) [reply getting-started/data-setup.rst:29]: OK.  How about the new words

Comment (timj) [reply getting-started/data-setup.rst:29]: Looks good. Thanks.

**Pull request merged**